src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/Dumpling.targets

```diff
@@ -37,6 +37,9 @@
       <CrashDumpFolder Condition="'$(CrashDumpFolder)' == '' and Exists('/cores/')">/cores/</CrashDumpFolder>
       <CrashDumpFolder Condition="'$(CrashDumpFolder)' == '' and '$(OS)' == 'Unix'">`pwd`</CrashDumpFolder>
     </PropertyGroup>
+	<ItemGroup>
```

Comment:
Nit;  You use tabs, the rest of the file uses spaces.

This looks like it'll work as long as the machines have pip, which most but not all python installs do have.  If we have to automate getting pip can discuss this later.